context/input.go

```diff
@@ -385,6 +388,8 @@ func (input *BeegoInput) Data() map[interface{}]interface{} {
 
```

Comment:
You also need to add lock for the Data function.

Reply:
I have added it.